src/components/DayOverview/DayOverviewButton.tsx

```diff
@@ -0,0 +1,63 @@
+"use client";
+
+import { NoSsr } from "@mui/material";
+import { PDFDownloadLink, PDFViewer } from "@react-pdf/renderer";
+import React from "react";
+import DayOverviewPdf from "@/components/DayOverview/DayOverviewPdf";
+import { ParcelOfSpecificDateLocation } from "@/components/DayOverview/DayOverview";
+import { Schema } from "@/supabase";
+
+interface DayOverviewerButtonProps {
+    date: Date;
+    location: string;
+    data: ParcelOfSpecificDateLocation[];
+    text: string;
+}
+
+const collectionCentreToAbbreviation = (
+    collectionCentre: Schema["parcels"]["collection_centre"]
+): string => {
+    switch (collectionCentre) {
+        case "Brixton Hill - Methodist Church":
+            return "BH_MC";
+        case "Clapham - St Stephens Church":
+            return "CLP_SC";
+        case "N&B - Emmanuel Church":
+            return "NAB_EC";
+        case "Streatham - Immanuel & St Andrew":
+            return "STM_IS";
+        case "Vauxhall Hope Church":
+            return "VHC";
+        case "Waterloo - Oasis":
+            return "WAT_OA";
+        case "Waterloo - St George the Martyr":
+            return "WAT_SG";
+        case "Waterloo - St Johns":
+            return "WAT_SJ";
+        case "Delivery":
+            return "Delivery";
+        default:
+            return "-";
```

Comment:
Do we want this as the default as opposed to an error message? Probably not needed tbf just something to think about

Reply:
Something like "Invalid Collection Centre" idk

Reply:
Let me consult Tom on this.
I assume the user using it will either be only granted privilege to access one location, or there is a drop down list to choose, so no matter what it shouldn't reach the default case
It's just if the program crashed or sth, should I let the program throw an error or return dash

Reply:
Yeah no that was what I was thinking, as it is something going wrong I think you should throw an error potentially

Reply:
Ok I changed it to throwing errors